addons/partner_autocomplete/static/src/js/partner_autocomplete_fieldchar.js

```diff
@@ -20,12 +20,7 @@ export class PartnerAutoCompleteCharField extends CharField {
     }
 
     async validateSearchTerm(request) {
-        if (this.props.name == 'vat') {
-            return this.partner_autocomplete.isVATNumber(request);
-        }
```

Comment:
We need to keep this condition, only suggestions based on VAT numbers should appear for the VAT field.
Instead, we should check that the value is either a VAT number or a GST number.